src/app/service-box/service-box.component.html

```diff
@@ -0,0 +1,66 @@
+<button mat-icon-button (click)="opened = !opened">
+	<mat-icon>apps</mat-icon>
+</button>
+
+
+<div class="box" [class.opened]="opened">
+	<div class="tip">
+		<div class="border"></div>
+		<div class="foreground"></div>
+	</div>
+
+	<div class="services">
+		<div class="row">
+			<a href="https://api.loklak.org/">
+				<div class="col">
+					<div class="img">
+						<img src="assets/images/cow_39x45.png" width="45" height="52" alt="Loklak Server">
+					</div>
+					<span>Loklak</span>
+				</div>
+			</a>
+			<a href="https://fossasia.org/">
+				<div class="col">
+					<div class="img">
+						<img src="assets/images/fossasia_logo_55x22.png" width="55" height="22" alt="Fossasia">
+					</div>
+					<span>FOSSASIA</span>
+				</div>
+			</a>
+			<a href="http://phimp.me">
```

Comment:
The links would load in the current tab, better to open them in a new tab.

Reply:
Mario wants it similar to google  and google services opens in that tab itself .